### ISSUE TITLE
Java version check needs fixing

### ISSUE THREAD

**Issue by manojsundaram** [cloudera-ps/prereq-checks#68]
Something is wrong with the Java version check and the script response needs a bit of tweaking as well. This is the output from a production environment.

 PASS  Java: Supported Oracle Java: /usr/java/jdk1.7.0_67-cloudera/bin/java
 PASS  Java: Unsupported Oracle Java: /usr/java/jdk1.6.0_31/bin/java
 PASS  Java: Supported Oracle Java: /usr/lib/jvm/jre-openjdk/bin/java
 PASS  Java: Unsupported Oracle Java: /usr/lib/jvm/jre-1.6.0-openjdk.x86_64/bin/java

ll /usr/lib/jvm/
total 8
drwxr-xr-x. 3 root root 4096 Sep 22  2016 java-1.6.0-openjdk-1.6.0.35.x86_64
drwxr-xr-x. 3 root root 4096 Sep 22  2016 java-1.7.0-openjdk-1.7.0.79.x86_64
lrwxrwxrwx. 1 root root   21 Sep 22  2016 jre -> /etc/alternatives/jre
lrwxrwxrwx. 1 root root   27 Sep 22  2016 jre-1.6.0 -> /etc/alternatives/jre_1.6.0
lrwxrwxrwx. 1 root root   38 Sep 22  2016 jre-1.6.0-openjdk.x86_64 -> java-1.6.0-openjdk-1.6.0.35.x86_64/jre
lrwxrwxrwx. 1 root root   27 Sep 22  2016 jre-1.7.0 -> /etc/alternatives/jre_1.7.0
lrwxrwxrwx. 1 root root   38 Sep 22  2016 jre-1.7.0-openjdk.x86_64 -> java-1.7.0-openjdk-1.7.0.79.x86_64/jre
lrwxrwxrwx. 1 root root   29 Sep 22  2016 jre-openjdk -> /etc/alternatives/jre_openjdk

**Comment by kmizumar** [comment]
```
PASS Java: Unsupported Oracle Java: /usr/java/jdk1.6.0_31/bin/java
PASS Java: Supported Oracle Java: /usr/lib/jvm/jre-openjdk/bin/java
PASS Java: Unsupported Oracle Java: /usr/lib/jvm/jre-1.6.0-openjdk.x86_64/bin/java
```

Can I see the output of `java -version` of the above java binaries?

**Comment by manojsundaram** [comment]
Maru, I dont have access to this cluster anymore. But may get access pretty soon. Will ping you

**Comment by jamestyj** [comment]
Should be fixed in Maru's commit 748e6c33d0c32222019add36b0889366a004e7fc. Closing.
